packages/dart_frog_auth/test/src/dart_frog_auth_test.dart

```diff
@@ -0,0 +1,206 @@
+// ignore_for_file: prefer_const_constructors
+import 'dart:io';
+
+import 'package:dart_frog/dart_frog.dart';
+import 'package:dart_frog_auth/dart_frog_auth.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:test/test.dart';
+
+class _MockRequestContext extends Mock implements RequestContext {}
+
+class _MockRequest extends Mock implements Request {}
+
+class User {
```

Comment:
Similarly to mocks, should also be private.

```suggestion
class _User {
```